src/Miso/Internal.hs

```diff
@@ -115,6 +115,17 @@ data ComponentState model action
   , componentActions    :: IORef (Seq action)
   }
 -----------------------------------------------------------------------------
+componentIds :: IORef [Int]
+{-# NOINLINE componentIds #-}
+componentIds = unsafePerformIO $ liftIO $ newIORef [ 1 .. ]
```

Comment:
Just curious: what's the benefit of having infinite list of IDs as opposed to a single Int value that gets incremented whenever freshComponentId is generated?

Reply:
Semantically equivalent, performance difference is negligible given the infrequency of usage